source/solvers/heat_transfer_assemblers.cc

```diff
@@ -422,18 +422,27 @@ HeatTransferAssemblerRobinBC<dim>::assemble_rhs(
   if (!scratch_data.is_boundary_cell)
     return;
 
-  auto &local_rhs = copy_data.local_rhs;
-
-
-  // Robin boundary condition, loop on faces (Newton's cooling law)
-  // implementation similar to deal.ii step-7
+  auto &       local_rhs = copy_data.local_rhs;
+  const double Stefan_Boltzmann_constant =
+    this->boundary_conditions_ht.Stefan_Boltzmann_constant;
+
+  // Robin boundary condition, loop on faces (Newton's cooling law +
+  // Stefan-Boltzmann law) Convection-radiation BC is a combination of
+  // convection and radiation. If the Stefan-Boltzmann constant (with default
+  // value = 0) is set to 0, only the convection component is considered,
+  // whereas if the convection coefficient, h (with default value = 0), is set
+  // to 0, only the radiation component is considered. Otherwise, both the
+  // convection and radiation are significant on the boundary implementation
```

Comment:
Should it be explicitly said what happens when both are zero?

Reply:
if both are 0, then using the convection-radiation BC is meaningless. I think it's not necessary to mention it.